src/components/SetupKeyNew.tsx

```diff
@@ -118,72 +115,79 @@ const SetupKeyNew = () => {
             revoked: formSetupKey.revoked,
             groupsToCreate: groupsToCreate,
             expires_in: expiresIn,
-            usage_limit: formSetupKey.usage_limit
-        } as SetupKeyToSave
-    }
+            usage_limit: formSetupKey.usage_limit,
+        } as SetupKeyToSave;
+    };
+
+    const handleFormSubmit = async () => {
+        try {
+            await form.validateFields();
+        } catch (e) {
+            const errorFields = (e as any).errorFields;
+            return console.log("errorInfo", errorFields);
```

Comment:
its ok, since console.log will just return void, we need to exit from function anyway and write log about errors, but if its more readable for team to be in 2 lines, its not a problem